.gitlab-ci.yml

```diff
@@ -340,7 +340,7 @@ doc:refman:dune:
   <<: *dune-ci-template
   variables:
     <<: *dune-ci-template-vars
-    DUNE_TARGET: refman-html
+    DUNE_TARGET: refman-html stdlib-html
```

Comment:
Any reason to do it in the same job vs adding another job?

Reply:
Efficiency. One documentation job seems enough IMO. In particular the deploy job needs to download this. That should change if we split the stdlib build off tho.

Reply:
I think you were right and I've changed this.